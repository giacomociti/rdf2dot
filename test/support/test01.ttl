@prefix v: <http://view/>.
@prefix attr:  <http://view/dot/attribute/>.

<urn:ann> a v:Node ;
    attr:label "Ann" .

<urn:bern> a v:Node ;
    attr:shape "rectangle" ;
    attr:label "bern" .

[] a v:Edge ;
    v:source <urn:ann> ;
    v:target <urn:bern> ;
    attr:label "lives in" ;
    attr:tooltip "urn:livesIn" .