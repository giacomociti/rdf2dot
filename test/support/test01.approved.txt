digraph {
  graph [rankdir="LR"]
  node [color="red"]
  edge [fontcolor="green",arrowhead="empty"]
  "urn:ann" [label="Ann"]
	"urn:bern" [label="bern",shape="rectangle"]
  "urn:ann" -> "urn:bern" [label="lives in",tooltip="urn:livesIn"]
}
